scripts/backend_tests.py

```diff
@@ -0,0 +1,331 @@
+# Copyright 2017 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Script for running backend tests in parallel.
+
+This should not be run directly. Instead, navigate to the oppia-ml/ folder and
+execute:
+
+    bash scripts/run_backend_tests.sh
```

Comment:
are `scripts/backend_tests.py` and `scripts/run_backend_tests.sh` exact copy of the scripts on Oppia or is there any modification that you have done?